presto-hive/src/main/java/io/prestosql/plugin/hive/HiveUpdatablePageSource.java

```diff
@@ -28,6 +28,7 @@
 import io.prestosql.spi.type.TypeManager;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.Path;
+import org.jetbrains.annotations.NotNull;
```

Comment:
This seems to be unused

Reply:
Yeah - I have a fixup for this. Just waiting for CI.